MAINTAINERS.yml

```diff
@@ -2339,6 +2339,40 @@ Xilinx Platforms:
   labels:
     - "platform: Xilinx"
 
+Infineon Platforms:
+  status: maintained
+  maintainers:
+    - ifyall
+  collaborators:
+    - npal-cy
+    - talih0
+    - billwatersiii
+    - sreeramIfx
+    - dey-r
+    - mcatee-infineon
+  files:
+    - boards/arm/nucleo_*/
```

Comment:
This isn't right; is it?

Reply:
Nope, that's copy-paste biting once again.